subprojects/gradle/build-verdict/src/main/kotlin/com/avito/android/build_verdict/TaskErrorOutputCaptureExecutionListener.kt

```diff
@@ -19,6 +19,11 @@ internal class TaskErrorOutputCaptureExecutionListener(
 
     override fun beforeExecute(task: Task) {
         when (task) {
+            is BuildVerdictTask -> {
```

Comment:
Why do we get a verdict before task execution?
Can't we do it after execution in case of failure?